README.md

```diff
@@ -187,6 +187,14 @@ GitHub github = Feign.builder()
                      .target(GitHub.class, "https://api.github.com");
 ```
 
+If you need to pre-process the response before give it to the Decoder, you can use the `mapAndDecode` builder method:
+
+```java
+GitHub github = Feign.builder()
+                     .mapAndDecode((response, type) -> myTransformingLogic(response, type), new GsonDecoder())
```

Comment:
I had 2 use cases for this, ciphered JSONS and JSONP responses...
I explained the second one because I think that could be more usual.